ice-phishing/src/agent.js

```diff
@@ -723,7 +900,9 @@ const provideHandleTransaction = (provider) => async (txEvent) => {
               if (!balance.eq(0)) return;
             }
           }
-          findings.push(createHighNumTransfersLowSeverityAlert(txFrom, transfersLowSeverity[txFrom]));
+          counters.detectedTransfersLow += 1;
```

Comment:
```suggestion
          counters.detectedTransfersLow += transfersLowSeverity[txFrom].length;
```
Suggestion to use the underlying amount again.